src/Discord.Net.Rest/Entities/Messages/RestMessage.cs

```diff
@@ -208,6 +211,12 @@ internal virtual void Update(Model model)
             else
                 _reactions = ImmutableArray.Create<RestReaction>();
         }
+        internal string SanitizeMessage()
```

Comment:
One thing that can be implemented is move this function to the [`MessageHelper`](https://github.com/Discord-Net-Labs/Discord.Net-Labs/blob/393d2d1e5f2ab51ed087fdd322ab2b3d78ec21ad/src/Discord.Net.Rest/Entities/Messages/MessageHelper.cs) class since its repeated in `SocketMessage`

Reply:
Updated in f3405b9